### ISSUE TITLE
Feature request - attach image as file

### ISSUE THREAD

**Issue by CiruxiaSWE** [v411e/hasswebhookbot#15]
Hi!

The plugin is working really well. But I am lacking one thing and that is to attach an image as a file instead.

An example is the use of camera.snapshot in Homeassistant, it saves the image as a file. I would like to ad that file as a payload instead of the Base64 string that is needed now for images.  

I have not yet found a way in homeassistant to encode camera.snapshot to a base64 string so the bot can accept it.

**Comment by v411e** [comment]
I agree, adding the image as a file would be much better than using a base64 string.
